### PR TITLE
PS-29 tags endpoint, crud and tests

### PR DESCRIPTION
**Pull request by noFrostoo**

### CODE REVIEW

File: services/backend/app/crud/crud_tags.py [hunk 1]
```diff
@@ -13,9 +15,47 @@ class CRUDTag(CRUDBase[Tag, TagCreate, TagUpdate]):
...
+    def update(
+        self, db: Session, *, db_obj: Tag, obj_in: Union[TagUpdate, Dict[str, Any]]
+    ) -> Tag:
+        if isinstance(obj_in, dict):
+            update_data = obj_in
+        else:
+            update_data = obj_in.dict(exclude_unset=True)
+        if update_data.get("name", False):
+            db_obj.name = update_data["name"]
+        if update_data.get("recipes", False):
```
Comment (lorlen) [inline services/backend/app/crud/crud_tags.py:44]: I'm not sure if it's a good idea for the `recipes` property to be modifiable.

Comment (noFrostoo) [reply services/backend/app/crud/crud_tags.py:44]: So you would only leave adding tags to recpie not recpie to tags?

Comment (lorlen) [reply services/backend/app/crud/crud_tags.py:44]: I don't really see any use for the latter. Tags are supposed to be immutable except for admins, and the UI probably will not have any way of adding recipes to tags.

Comment (noFrostoo) [reply services/backend/app/crud/crud_tags.py:44]: Fair

**Review by lorlen — APPROVED** [review]

**Pull request merged**